wetterdienst/provider/dwd/observation/metaindex.py

```diff
@@ -158,19 +159,10 @@ def _read_meta_df(file: BytesIO) -> pd.DataFrame:
     """
     df = pd.read_fwf(
         filepath_or_buffer=file,
-        colspecs=[
-            (0, 5),
-            (5, 14),
-            (14, 23),
-            (23, 38),
-            (38, 50),
-            (50, 60),
-            (60, 102),
-            (102, 200),
-        ],
+        colspecs="infer",
```

Comment:
Maybe add `infer_nrows=np.Infinity` here as well? See e622a4a.

Reply:
Didn't work, fields seem to be undetectable, I reset it!